exercises/giovanni_disiena/ex02_oo_basics/src/Rectangle.h

```diff
@@ -0,0 +1,21 @@
+#ifndef RECTANGLE_H
+#define RECTANGLE_H
+
+#include "ShapeBase.h"
+
+class Rectangle : public ShapeBase {
+public:
+  // constructs Rectangle object
+  Rectangle(double, double);
```

Comment:
It is good practice to include the name of the variables, it makes it much easier to read and use.

Reply:
Similarly for all the shapes